content/momentum/changelog/4/4-5-0.md

```diff
@@ -0,0 +1,70 @@
+---
+lastUpdated: "11/15/2022"
+title: "Momentum 4.5.0 Changelog"
+description: "Momentum 4.5.0 was released on 2022-12-01. This section will list all of the major changes that happened with the release of Momentum 4.5.0 Depending on installation type, all changes may not be applicable"
+---
+
+**Momentum 4.5.0** was released on 2022-12-01. This section will list all of the major changes that happened with the release of **Momentum 4.5.0**. Depending on installation type, all changes may not be applicable
+
+<a name="changelog.4.5.0.table"></a>
+
+| Type | Ticket | Description |
+| --- | --- | --- |
+| Fix | SD-2935 | Crash when outbound_throttle_messages is configured at the same time as core_mailq_throttle_message hook |
+| Fix | SD-2995 | soft_bounce functionality periodically tries using an invalid connection handle causing a crash |
+| Fix | SD-3167 | Stack overflow in modules/delivery/http2/http2clnt.c http_send |
+| Fix | SD-3259 | Momentum incorrectly interprets checkpoint file data on restart |
+| Fix | SD-3263 | DKIM Oversigning to Avoid Replay Attacks |
+| Fix | SD-3288 | Transmission API does not sanity check recipient list for empty array |
+| Fix | SD-3335 | lu_pull not working on AL2 nodes |
+| Fix | SD-3374 | lu_pull not working on AL2 nodes, round 2 |
+| Fix | SD-3497 | DSNs are not Generated when RCPTO is Upper Case |
+| Fix | SD-3535 | extract_sender_from_msg moved to msys namespace |
+| Fix | SD-3606 | rcpt_to incorrect for "Unconfigured sending domain" rejection events for multi-recipient SMTP injections |
+| Fix | SD-3619 | Fix crashes in Sieve when creating new messages |
+| Fix | SD-3649 | Upgrade to latest jlog package |
+| Fix | SD-3685 | Invalid write in outbound smtp auth when base64 encoded username + password is length of a power of 2 |
+| Fix | SD-3702 | Incorrect DKIM signature for some large body messages |
+| Security Fix | SD-2871 | Fix Error "Inconsistent nullness check" |
```

Comment:
If there is a wording we would like better for these CodeQL related fixes, let me know!

Reply:
Do we need to keep the description exactly the same as in Jira or can we customise?

Reply:
We can customize as we see fit here.

Reply:
Cool, I don't really know a good way to describe these CodeQL fixes, but I think we should stick to a pattern like:

- ... alert
- ... alert in Momentum

Reply:
I was specifically referring to the `Security Fix` type. We can make that whatever we want, but I wasn't sure if that was the best way to put it or not 😅